src/iologindata.cpp

```diff
@@ -400,6 +400,8 @@ bool IOLoginData::loadPlayer(Player* player, DBResult_ptr result)
 		player->skills[i].percent = Player::getPercentLevel(skillTries, nextSkillTries);
 	}
 
+	player->direction = static_cast<Direction> (result->getNumber<uint16_t>("direction"));
```

Comment:
Isn't 8 bits enough for direction?

Reply:
I think could be enough, but I was based on lines like that https://github.com/otland/forgottenserver/blob/master/src/iologindata.cpp#L364 which could be also int8 (which already is in https://github.com/otland/forgottenserver/blob/master/src/position.h#L42). Should I change on `player->loginPosition.z` also?

Reply:
I think could be enough, but I was based on lines like that https://github.com/otland/forgottenserver/blob/master/src/iologindata.cpp#L364 which could be also int8 (which already is in https://github.com/otland/forgottenserver/blob/master/src/position.h#L42). Should I change on `player->loginPosition.z` also?

Reply:
As Mark said here https://github.com/otland/forgottenserver/pull/2532#discussion_r260477881 it should remain int16.